portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,62 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+// import com.google.sps.data.ServerStats;
+import com.google.gson.Gson;
+
+/** Servlet that handles comments data.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private ArrayList<String> comments;
```

Comment:
I would recommend having a class for Comment. Even if for now it only contains a single string it would make it easier to expand later

Reply:
I will work on that.